### ISSUE TITLE
Ragged blue muscle fibers

### ISSUE THREAD

**Issue by MLK6014** [obophenotype/human-phenotype-ontology#6371]
**Preferred term label:**
Ragged blue muscle fibers

**Synonyms**
Ragged blue fibers

**Definition (free text, please give PubMed ID)**
A characterisitic histological abnormality, like the Rageed Red Fibers (RRF) the same fibres are called ‘ragged blue’ fibers because they show an intense, irregular staining with Succinic Dehydrogenase and NADH histochemical stains

**Parent term (use hpo.jax.org/app)**
Abnormal muscle fiber morphology HP:0004303

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Mitochondrial disease

**Your nano-attribution (ORCID)**

**Comment by MLK6014** [comment]
after discussion with Professor of brain and mitochondrial research at Murdoch childrens research institute **Muscle abnormality related to mitochondrial dysfunction HP:0003800**
seems a better suited parent concept.  thanks MLK6014

**Comment by pnrobinson** [comment]
So is this actually a synonym for ragged red fiber -- basically the same abnormality shown with a different stain?
https://neuropathology-web.org/chapter13/chapter13dMetabolic.html

**Comment by MLK6014** [comment]
Ok thank you for your feedback, can we please request the addition of a synonym "Ragged blue muscle fibers" to the existing HPO term Ragged-red muscle fibers HP:0003200. Should I do this through a separate issue request for a new synonym?
